auth0/v3/management/users_by_email.py

```diff
@@ -0,0 +1,44 @@
+from .rest import RestClient
+
+
+class UsersByEmail(object):
+
+    """Auth0 users by email endpoints
+
+    Args:
+        domain (str): Your Auth0 domain, e.g: 'username.auth0.com'
+
+        token (str): Management API v2 Token
+
+        telemetry (bool, optional): Enable or disable Telemetry
+            (defaults to True)
+    """
+
+    def __init__(self, domain, token, telemetry=True):
+        self.domain = domain
+        self.client = RestClient(jwt=token, telemetry=telemetry)
+
+    def _url(self):
+        url = 'https://%s/api/v2/users-by-email' % self.domain
+        return url
+
+    def search_users_by_email(self, email, fields=[], include_fields=True):
```

Comment:
> Default values are evaluated when the function is compiled, which is an important distinction if the value is mutable.

Please change `[]` to `None` and do the check later. See how it's done [here](https://github.com/auth0/auth0-python/blob/cbce31b770619a772cfecc5fea6df4390e6e95cc/auth0/v3/management/logs.py#L63)